### ISSUE TITLE
Windows Requirements for usage

### ISSUE THREAD

**Issue by Selena-Gomez** [randoms/burst-link#19]
After encountering this error
![msvcr120dll](https://cloud.githubusercontent.com/assets/1886075/5812899/06a28dc6-a031-11e4-893e-8f4cbab83c48.png)
I learned that you need to download the C++ Redistributable Packages for Visual Studio 2013, found [here](http://www.microsoft.com/en-ca/download/details.aspx?id=40784). Specifically the x86 version.

**Comment by randoms** [comment]
Yes, the package is needed. I'd better add MSVCR120.dll in the release directory. Glad see your feedback.
